ambrs/visualization/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Refactored visualization subpackage for AMBRS.
+
+Legacy ridge/FastGrid plotting has been replaced by PyParticle-based helpers.
+Only the new PyParticle line/grid comparison utilities are exported.
+
+If legacy functions are still needed temporarily, import from
+`ambrs.visualization_legacy.output_plot_legacy` explicitly.
```

Comment:
I don't see a `visualization_legacy` folder in this PR. Does it need to be added?